altimeter_ms5607_spi/altimeter_ms5607_spi.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <math.h>
+
+#if defined(STM32F1)
+#include "stm32f1xx_hal.h"
+#endif
+
+#ifndef SERIAL_TIMEOUT
+#define SERIAL_TIMEOUT 10
+#endif
+
+class AltimeterMs5607Spi {
```

Comment:
Also, can we just name these classes simply like `Altimeter` and exclude the other details? It makes the code everywhere more concise, and we aren't going to have multiple drivers for the same device type with different part numbers or communication protocols in use at the same time. LMK what you think @DanielHeEGG

Reply:
Even though we don't have anything planned for the moment, there is a possibility of having multiple sensors that do the same thing but from different brands to ensure redundancy and have different failure modes. I'd like to keep that option open